src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/CSharp/CSharpProjectedDocument.ts

```diff
@@ -39,7 +39,7 @@ export class CSharpProjectedDocument implements IProjectedDocument {
         }
 
         let content = this.content;
-        for (const edit of edits) {
+        for (const edit of edits.reverse()) {
```

Comment:
Why does reversing the edit sequence resolve it?

Reply:
Consider the following set of edits,
1: Insert `a` at position 0
2: Insert `b` at position 1

Let's say the existing text is "Hello". Now when we apply the above set of edits in parallel (which is what VS and Roslyn does),
we would get, `aHbello` (this is the correct answer)

Now instead if we apply it one by one,
Apply edit 1: `aHello`
Apply edit 2: `abHello` (this is wrong)

If we apply it in reverse,
Apply edit 2: `Hbello`
Apply edit 1: `aHbello`

Hope this example helped.

Reply:
Gotcha, thanks!